src/vulkan/vertex_buffer_test.cc

```diff
@@ -280,6 +288,7 @@ TEST_F(VertexBufferTest, R16G11B10_SFLOAT) {
   values[2].SetDoubleValue(0.1171875);
 
   auto format = MakeUnique<Format>();
+  format->SetFormatType(FormatType::kR32G32_SFLOAT);  // big enough to cover.
```

Comment:
Needs a B component.

Reply:
These don't actually match the components. We purposely make a weird component below. I just have to have a format for things to work correctly. Since there is no matching format, I made one that is at least close in byte size.